# This message contains status data from motor.

float32 position
float32 velocity
float32 current
int8 temperature
int8 motor_fault